src/pages/index.tsx

```diff
@@ -35,6 +70,23 @@ const Index: FC = () => {
     }
   }, [isLoading, response]);
 
+  useEffect(() => {
+    const url = SELF_URL;
+    const fetchData = async () => {
+      try {
+        const r_esponse = await fetch(url);
+        const json = await r_esponse.json();
+        if (json.user.roles.admin && json.user.roles.super_user === true) {
```

Comment:
I believe you need OR here.
Also, we don't need `===` for boolean values

Reply:
We can use OR here if we want user to be either admin or super_user but I used AND to include only those users which have both. Thinking only Ankush has both rights, I used AND. Should I change it to OR

Reply:
About === the code shows eslint issue whenever I try to use ==.
I know that === is used for type checking and value checking and == is just to check whether values are equal but it shows 

Expected '===' and instead saw '=='.eslinteqeqeq

Reply:
Loose the `if` and make it `setRoles(adminUser || superUser)`

Reply:
Also, I don't see any `roles` field [here](https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users#user-object) and [here](https://github.com/Real-Dev-Squad/website-backend/blob/605fcec70157a760a165fb212f22758c9e0fd2b5/docs/swaggerDefinition.js#L412) where did you find the `roles` field.

Reply:
I used the SELF_URL and fetched it. This gives the user details (username, GitHu_id etc and roles also)

Reply:
We have API-Contracts and that should be up to date; in case it's not, you should make the concerned person / AO aware of it.
I'll make an issue to update the API Contracts 👍🏽

Reply:
Okay